### DISCUSSION TITLE
Uncaught TypeError: Cannot set property 'value' of undefined

### DISCUSSION THREAD

**Discussion by furiousnur** [livewire/livewire#2134] [1. Help]
Uncaught TypeError: Cannot set property 'value' of undefined
    at HTMLInputElement.clearFileInputValue (**DisableForms.js:21**)

My 21 number line is:  if (node.hasAttribute('wire:ignore')) return false

**Here is my DisableForms.js file code**

import store from '@/Store'
import { wireDirectives } from '../util'

let cleanupStackByComponentId = {}

export default function () {
    store.registerHook('element.initialized', (el, component) => {
        let directives = wireDirectives(el)

        if (directives.missing('submit')) return

        // Set a forms "disabled" state on inputs and buttons.
        // Livewire will clean it all up automatically when the form
        // submission returns and the new DOM lacks these additions.
        el.addEventListener('submit', () => {
            cleanupStackByComponentId[component.id] = []

            component.walk(node => {
                if (! el.contains(node)) return

                if (node.hasAttribute('wire:ignore')) return false

                if (
                    // <button type="submit">
                    (node.tagName.toLowerCase() === 'button' &&
                        node.type === 'submit') ||
                    // <select>
                    node.tagName.toLowerCase() === 'select' ||
                    // <input type="checkbox|radio">
                    (node.tagName.toLowerCase() === 'input' &&
                        (node.type === 'checkbox' || node.type === 'radio'))
                ) {
                    if (!node.disabled)
                        cleanupStackByComponentId[component.id].push(
                            () => (node.disabled = false)
                        )

                    node.disabled = true
                } else if (
                    // <input type="text">
                    node.tagName.toLowerCase() === 'input' ||
                    // <textarea>
                    node.tagName.toLowerCase() === 'textarea'
                ) {
                    if (!node.readOnly)
                        cleanupStackByComponentId[component.id].push(
                            () => (node.readOnly = false)
                        )

                    node.readOnly = true
                }
            })
        })
    })

    store.registerHook('message.failed', (message, component) => cleanup(component))
    store.registerHook('message.received', (message, component) => cleanup(component))
}

function cleanup(component) {
    if (!cleanupStackByComponentId[component.id]) return

    while (cleanupStackByComponentId[component.id].length > 0) {
        cleanupStackByComponentId[component.id].shift()()
    }
}

How can I solve this issue..?

**Comment by joshhanley** [comment]
@furiousnur there was a bug with the Livewire file uploads introduced in 2.3.2 that is fixed in the latest version 2.3.5. Maybe that is the error that you were getting.

See PR here #2055

Can you upgrade to the latest version and see if the issue is still happening?
